addons/portal/__manifest__.py

```diff
@@ -39,18 +39,71 @@
         'web.assets_frontend': [
             'portal/static/src/scss/portal.scss',
             'portal/static/src/js/portal.js',
-            'portal/static/src/js/portal_chatter.js',
             'portal/static/src/xml/portal_chatter.xml',
             'portal/static/src/js/portal_composer.js',
             'portal/static/src/js/portal_security.js',
             'portal/static/src/js/portal_sidebar.js',
             'portal/static/src/xml/portal_security.xml',
             'portal/static/src/js/components/**/*',
             'portal/static/src/signature_form/**/*',
+
+            ('include', 'portal.assets_chatter'),
+            'portal/static/src/chatter/core/**/*',
+            'portal/static/src/chatter/frontend/**/*',
         ],
         'web.assets_tests': [
             'portal/static/tests/**/*',
         ],
+        'portal.assets_chatter': [
+            'web/static/src/views/view_dialogs/form_view_dialog.js',
+            'web/static/src/core/debug/*',
+            'web/static/src/core/commands/*',
+            'web/static/src/model/**/*',
+            'web/static/src/search/**/*',
+            'web/static/src/views/view.js',
+            'web/static/src/views/view_hook.js',
+            'web/static/src/views/onboarding_banner.js',
+            'web/static/src/webclient/actions/action_service.js',
+            'web/static/src/webclient/actions/action_hook.js',
+            'web/static/src/webclient/actions/action_dialog.js',
+            'web/static/src/webclient/actions/reports/utils.js',
+            'web/static/src/webclient/actions/reports/report_action.js',
+            'web/static/src/webclient/actions/reports/report_hook.js',
+            'web/static/src/views/utils.js',
+            'web/static/src/views/fields/formatters.js',
+            'web/static/src/views/fields/file_handler.*',
+
+            'mail/static/src/model/**/*',
+            'mail/static/src/core/common/**/*',
+            'mail/static/src/core/web_portal/**/*',
+            'mail/static/src/utils/common/**/*',
+            'mail/static/src/chatter/web_portal/**/*',
+            'mail/static/src/discuss/typing/**/*',
+            'mail/static/src/discuss/core/common/action_panel.*',
+        ],
+        'portal.assets_embed': [
```

Comment:
What is embed portal?

Reply:
The bundle that provides the shadow dom's requirements.